app/src/main/java/com/poupa/vinylmusicplayer/glide/artistimage/ArtistImageFetcher.java

```diff
@@ -68,22 +71,36 @@ public void onResponse(@NonNull Call<DeezerResponse> call, @NonNull Response<Dee
 
                         try {
                             DeezerResponse deezerResponse = response.body();
-                            String url = deezerResponse.getData().get(0).getPictureMedium();
-                            streamFetcher = new OkHttpStreamFetcher(okhttp, new GlideUrl(url));
-                            streamFetcher.loadData(priority, callback);
+                            List<Data> data = deezerResponse.getData();
+                            if (data.size() > 0) {
+                                String url = data.get(0).getPictureMedium();
+
+                                // Fragile way to detect a place holder image returned from Deezer:
+                                // ex: "https://e-cdns-images.dzcdn.net/images/artist//250x250-000000-80-0-0.jpg"
+                                // the double slash implies no artist identified
+                                final boolean placeholderUrl = url.contains("/images/artist//");
```

Comment:
maybe we can simply detect if there are more than one occurrence of `//`?

Reply:
@paolovalerdi Do you have any comment on this, any better way to detect the place holder image? (Seems that I cannot add you a a reviewer).
@AdrienPoupa that would certainly work, but the detection remains fragile.